src/protocol/util/AsyncEventServer.h

```diff
@@ -38,7 +38,13 @@ class AsyncEventServer
   public:
     explicit AsyncEventServer(std::shared_ptr<boost::asio::io_context>    io_context,
                               const protocol_strategy_factory<char>::ptr& protocol,
+                              const std::string&                          host,
                               unsigned short                              port);
+
+    AsyncEventServer(std::shared_ptr<boost::asio::io_context>             io_context,
```

Comment:
I dont think this constructor without the host is being used or is helpful, and could be removed